src/reconcile/rec_visibility.c

```diff
@@ -134,30 +234,243 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
     upd_select->upd = NULL;
     upd_select->upd_saved = false;
 
+    cache = S2C(session)->cache;
+    las_cursor = NULL;
+    cbt = &r->update_modify_cbt;
+    __wt_modify_vector_init(&modifies, session);
```

Comment:
I wrote it like that because this function is basically a constructor for the vector. So it seems more logical to me to have the object first. I don't really mind though so I'm happy to change it if you're unconvinced by that explanation.

Reply:
Yeah, `session` (or a cursor delivering a session), should always come first.

Reply:
24d9694